src/parser/jobs/sch/modules/ScholarCooldownDowntime.tsx

```diff
@@ -0,0 +1,23 @@
+import {Trans} from '@lingui/react'
+import {ActionLink} from 'components/ui/DbLink'
+import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
+import React, {Fragment} from 'react'
+
+export default class ScholarCooldownDowntime extends CooldownDowntime {
```

Comment:
We don't allow default exports.

```suggestion
export class CooldownDowntime extends CoreCooldownDowntime {
```